src/components/Panel/Panel.tsx

```diff
@@ -112,7 +130,14 @@ export class Panel extends BaseComponent<IPanelProps, IPanelState> {
               isDarkThemed={ false }
               onClick={ isLightDismiss ? this._onPanelClick : null }
               />
-            <div className='ms-Panel-main'>
+            <FocusTrapZone
+              className='ms-Panel-main'
+              elementToFocusOnDismiss={ elementToFocusOnDismiss }
+              isClickableOutsideFocusTrap = { isLightDismiss }
+              ignoreExternalFocusing={ ignoreExternalFocusing }
+              forceFocusInsideTrap={ forceFocusInsideTrap }
+              firstFocusableSelector={ firstFocusableSelector }
+            >
```

Comment:
I am not sure it makes sense to add all of these to the props via `IAccessiblePopupProps`. Instead of that, could you just explicitly add the specific props you'll pass through and omit others? I think `IAccessiblePopupProps.ts` should be deleted as it's just making a black box that we're adding to a component's API surface. Pick the things that make sense or are required to pass through and add those to the contract only.